checker/dns/cname/cname_test.go

```diff
@@ -22,15 +22,15 @@ import (
 	"wait4x.dev/v2/checker"
 )
 
-const server = "wait4x.dev"
+const server = "www.company.info"
```

Comment:
Why company.info?

Reply:
Since we don't have a CNAME for "wait4x.dev," I'll update it with a related domain.